javav2/usecases/creating_redshift_application/Readme.md

```diff
@@ -0,0 +1,1199 @@
+
+**Note**: Amazon Redshift is a fully managed, petabyte-scale data warehouse service in the cloud. You can start with just a few hundred gigabytes of data and scale to a petabyte or more. This enables you to use your data to acquire new insights for your business and customers. For more information, see [What is Amazon Redshift?](https://docs.aws.amazon.com/redshift/latest/mgmt/welcome.html).
+
+The data is stored in a Redshift table named **blog**, as shown in this illustration. 
+
+![AWS Tracking Application](images/Redshift.png)
+
+The **blog** table contains these fields: 
+
+- **idblog**- a varchar field that stores a GUID value and represents the PK.
+- **date**- a date field that represents the date when the record was added. 
+- **title**- a varchar field that represents the title. 
+- **body**- a varchar field that represents the body. 
+- **author**-a varchar field that represents the author. 
+
+**Note**: For more information about supported field data types, see [Data types](https://docs.aws.amazon.com/redshift/latest/dg/c_Supported_data_types.html). 
+
+The application you create uses Spring Boot APIs to build a model, different views, and a controller. This web application also lets a user submit a new job posting that is then stored into the **blog** table, as shown in this illustation. 
+
+![AWS Tracking Application](images/NewRecorda.png)
+
+This example application lets you view the posts by choosing the **Get Posts** menu item and choosing one of the available buttons. For example, you can view five recent posts by choosing the **Five Posts** button, as shown in the following illustration.
+
+![AWS Tracking Application](images/FiveRecordsa.png)
+
+This application also supports viewing the result set in different languages. For example, if a user wants to view the result set in Spanish, they can choose Spanish from the dropdown field and the data is translated to the given language by using the Amazon Translate Service, as shown in this illustration. 
+
+![AWS Tracking Application](images/Spanisha.png)
+
+**Cost to complete:** The AWS services included in this document are included in the [AWS Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc).
+
+**Note:** Be sure to terminate all of the resources you create while going through this tutorial to ensure that you’re not charged.
+
+#### Topics
+
++ Prerequisites
++ Create an IntelliJ project 
++ Add the POM dependencies to your project
++ Set up the Java packages in your project
++ Create the Java classes
++ Create the HTML files
++ Package the application into a JAR file
++ Deploy the application to Elastic Beanstalk
+
```

Comment:
(optional) this could be a numbered list because the steps must be done in order.

Reply:
done